scripts/download_image.py

```diff
@@ -295,13 +295,10 @@ def accept_game_download_warning(workers: int,
     False
     """
     import textwrap
-    import shutil
-    choice = input_fn('\n'.join(textwrap.wrap(
-            textwrap.dedent(
-                    WARNING_MESSAGE.format(workers, GAME_WORKER_LIMIT)),
-            shutil.get_terminal_size().columns))).lower()
-    while choice not in ['y', 'n']:
-        choice = input_fn('Either enter Y or N as an argument:').lower()
+    choice = input_fn('\n'.join(textwrap.wrap(textwrap.dedent(
+        WARNING_MESSAGE.format(workers, GAME_WORKER_LIMIT)))))
+    while choice.lower() not in ['y', 'n']:
+        choice = input_fn('Either enter Y or N as an argument:')
     return choice == 'y'
```

Comment:
can you revert here, my bug (choice may be 'Y') upper letter
or return choice in "Yy" ... strange though.  
or return choice in {'Y', 'y'}  # set-iteral

Reply:
Fixed return choice.lower() == 'y'